src/sentry/tasks/summaries/utils.py

```diff
@@ -150,48 +150,34 @@ def project_key_performance_issues(ctx, project):
 
         # Fine grained query for 3 most frequent events happend during last week
         query = Query(
-            match=Entity("transactions"),
+            match=Entity("search_issues"),
             select=[
-                Column("group_ids"),
+                Column("group_id"),
                 Function("count", []),
             ],
             where=[
-                Condition(Column("finish_ts"), Op.GTE, ctx.start),
-                Condition(Column("finish_ts"), Op.LT, ctx.end + timedelta(days=1)),
-                # transactions.group_ids is a list of group_ids that the transaction was associated with.
-                # We want to find the transactions associated with group_id_to_group.keys()
-                # That means group_ids must intersect with group_id_to_group.keys() in order for the transaction to be counted.
-                Condition(
-                    Function(
-                        "notEmpty",
-                        [
-                            Function(
-                                "arrayIntersect",
-                                [Column("group_ids"), list(group_id_to_group.keys())],
-                            )
-                        ],
-                    ),
-                    Op.EQ,
-                    1,
-                ),
```

Comment:
Don't we still need to filter on the group id keys here? Something like `Condition(Column("group_id"), Op.IN, list(group_id_to_group.keys())